OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -510,39 +515,58 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards covered with reddish-brown <material key="leather"/>.
-                                    The spine is further covered with a reddish-brown tooled <material key="leather"/> overback.
-                                </decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                                    <material key="wood"/>       
-                                    <material key="leather"/>     
-                                </decoNote>
-                                <decoNote xml:id="b3" type="Cover" color="reddish-brown">Blind-tooling decoration on the overback:
-                                    double lines, double circles.</decoNote>
-                                <decoNote xml:id="b4" type="SewingStations">4</decoNote>
-                                <decoNote xml:id="b5" type="Spine">The stitches of four endband chains are sewn into the leather of the spine, two being close to the head and two to the tail.</decoNote>
-                                <decoNote xml:id="b6" type="Other">A green <term key="leafTabMark">leaf tab marker</term> for navigating is inserted on the upper corner of 
-                                    <locus target="#26"/>, incipit page of the reading for Tuesday.</decoNote>
+                            <binding contemporary="true Ethiopian">
+                              <decoNote xml:id="b1" type="bindingMaterial">
+                                The book is bound in <material key="wood">wooden</material> boards. An older blind-tooled <material
+                                key="leather">leather</material>, perhaps contemporary with the majority of manuscript's content, covers them entirely and a new layer has been added on top, which covers
+                                only the spine and the back edge of the boards.
+                              </decoNote>
+                              <decoNote xml:id="b2" type="Boards">
+                                The boards are flush with the bookblock and have internal shallow bevels.
+                              </decoNote>
+                              <decoNote xml:id="b3" type="Cover" color="reddish-brown">
+                                The former blind tooled cover is preserved, despite deteriorated, under the present one. The decoration is partially
+                                legible on the edges, where a pattern of double circles is found.
+                                The present leather is blind-tooled as well but covers only the spine and partially the boards.
+                                On its surface bands of two <term key="singleStraightLine">single straight lines</term> intersect
+                                forming a column of juxtaposed panels. Within each panel, the lines cross each other forming a crux decussata and at the intersections of the lines,
```

Comment:
Also here `<foreign xml:lang="la">crux decussata</foreign>`, what do you think?

Reply:
You are right! @gnizla I wonder if you are using different term to describe it

Reply:
It's not my area, but crux decussata is fine!